frontend/src/views/PdfReportView.vue

```diff
@@ -362,549 +362,504 @@ import PdfPreviewDialog from "@/components/pdfreport/assetforms/PdfPreviewDialog
-
-@Component({
-    components: {
-        PdfPreviewDialog,
-        draggable,
-        DisplayImageAsset,
-        ImageAssetForm,
-        HeadingAssetForm,
-        TextAssetForm,
-        DatatableAssetForm,
-        DeleteDialog,
-    },
-})
-export default class PdfReportView extends Vue {
-    // mouse events
-    clickable = 0;
-    draggable = false;
-    dragging = false;
-
-    // Formulare
-    editImage = false;
-    editHeading = false;
-    editText = false;
-    editDatatable = false;
-    deleteDialog = false;
-    previewPdfDialog = false;
-
-    // die einzelnen Assets
-    imageAsset: ImageAsset = new ImageAsset("", "");
-    headingAsset: HeadingAsset = new HeadingAsset("", AssetTypesEnum.HEADING1);
-    textAsset: TextAsset = new TextAsset("");
-    datatableAsset: DatatableAsset = new DatatableAsset(
-        {} as OptionsDTO,
-        "",
-        ""
+import { useStore } from "@/api/util/useStore";
+import { useI18n } from "vue-i18n-composable";
+
+const clickable = ref(0);
+const isDraggable = ref(false);
+const dragging = ref(false);
+
+const editImage = ref(false);
+const editHeading = ref(false);
+const editText = ref(false);
+const editDatatable = ref(false);
+const deleteDialog = ref(false);
+const previewPdfDialog = ref(false);
+
+const imageAsset = ref<ImageAsset>(new ImageAsset("", ""));
+const headingAsset = ref<HeadingAsset>(
+    new HeadingAsset("", AssetTypesEnum.HEADING1)
+);
+const textAsset = ref<TextAsset>(new TextAsset(""));
+const datatableAsset = ref<DatatableAsset>(
+    new DatatableAsset({} as OptionsDTO, "", "")
+);
+
+const assetId = ref(0);
+const loadingPdf = ref(false);
+
+const editables = ref([
+    AssetTypesEnum.TEXT,
+    AssetTypesEnum.IMAGE,
+    AssetTypesEnum.HEADING1,
+    AssetTypesEnum.HEADING2,
+    AssetTypesEnum.HEADING3,
+    AssetTypesEnum.HEADING4,
+    AssetTypesEnum.HEADING5,
+    AssetTypesEnum.DATATABLE,
+    AssetTypesEnum.DATATABLE_MESSSTELLE,
+]);
+
+const fab = ref(false);
+const pdfSourceAsBlob = ref<Blob>(new Blob());
+const pdfSourceForPreview = ref<Uint8Array>(new Uint8Array());
+const assets = ref<BaseAsset[]>([]);
+const store = useStore();
+const i18n = useI18n();
+
+onMounted(() => {
+    assets.value = assetsFromStore();
+    if (!store.getters.hasTitlePage) {
+        createFirstPage();
+    }
+});
+
+const fabColor = computed(() => {
+    return fab.value ? "grey darken-1" : "secondary";
+});
+
+/**
+ * Erstellt die Titelseite des PDF Reports.
+ */
+function createFirstPage(): void {
+    // Damit die FirstPage immer oben steht wenn die Seite das erste mal angesurft wird aber schon Assets dem Report hinzugefügt wurden:
+    // Assets reversen, Titelpage in umgekehrter Reihenfolge hinzufügen und Assets nochmals reversen
+    assets.value.reverse();
+    // Seitenumbruch
+    assets.value.push(new PagebreakAsset());
+    // Datum
+    save(
+        new HeadingAsset(
+            "erstellt am " + i18n.d(new Date(), "short", "de-DE"),
```

Comment:
Dafür gibt es in den DateUtils eine eigene Methode. Bitte darauf umstellen

Reply:
In diesem Zuge bitte auch einfach die DateUtils umstellen auf das composable von i18n